MyMovies/View Controllers/MovieSearchTableViewController.swift

```diff
@@ -52,7 +53,30 @@ class MovieSearchTableViewController: UITableViewController {
         cell.textLabel?.text = movieController.searchedMovies[indexPath.row].title
         return cell
    }
-}
+    
+     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+            if let indexPaths = tableView.indexPathsForSelectedRows {
+                for indexPath in indexPaths {
+                    let movieDBMovie = movieController.searchedMovies[indexPath.row]
+
+                    let movie = Movie(title: movieDBMovie.title, hasWatched: false)
+                    movieController.sendMovieToServer(movie: movie)
+                    
+                    do {
+                        try CoreDataStack.shared.mainContext.save()
+                        navigationController?.dismiss(animated: true, completion: nil)
```

Comment:
I really like how you added this here. In my personal opinion I like that it dismissed when I select a movie to add. As a user it confirms to me that I selected and saved a movie I want to watch.